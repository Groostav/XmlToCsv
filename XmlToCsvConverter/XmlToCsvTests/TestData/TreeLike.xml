<Node>
  <Name>Everard</Name>
  <Node>
    <Name>Bob</Name>
  </Node>
  <Node>
    <Name>Ken</Name>
  </Node>
  <Node>
    <Name>James</Name>
    <Node>
      <Name>Molly</Name>
    </Node>
    <Node>
      <Name>Betty</Name>
    </Node>
    <Node>
      <Name>Angela</Name>
    </Node>
  </Node>
</Node>