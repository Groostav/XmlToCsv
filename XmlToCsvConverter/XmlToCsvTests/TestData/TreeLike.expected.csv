Name,Node_Id,Node_Id_0
"Everard",0,
"Bob",1,0
"Ken",2,0
"James",3,0
"Molly",4,3
"Betty",5,3
"Angela",6,3
